package.json

```diff
@@ -5,11 +5,12 @@
   "main": "index.js",
   "scripts": {
     "bundle": "bundle-dependencies update",
+    "pretest": "npm run clean && npm run instrument",
```

Comment:
Even if we don't go with the rest of this change it would be nice to split out the `pretest` script.